### PR TITLE
 #165273465 Configure Postgresql as DB Engine

### PR DESCRIPTION
**Pull request by 3Nakajugo**
#### What does this PR do?
Configures Postgresql as Database Engine.
#### Description of Task to be completed?
- update settings with PostgreSQL engine
- install psycopg2
- update requirements.txt
- run migrations.
#### How should this be manually tested?
- create a .env file with the following variables.
```
export DB_NAME=value
export DB_USER=value
export DB_PASS=value
export DB_HOST=value
export DB_PORT=value
```
- make migrations 
`python manage.py migrate`
- run server
`python manage.py runserver`
#### What are the relevant pivotal tracker stories?
[#165273465](https://www.pivotaltracker.com/story/show/165273465)
#### Screenshots (if appropriate)
- Register user
image
- User saved in Database
image

#### Reviewers
@mwinel @engjames @sanya-kenneth @joelmugaya @KabohaJeanMark

### CODE REVIEW

**Review by mwinel — APPROVED** [review]
Good job edna. Why can't it be deployed.

**Comment by archibishop** [comment]
@3Nakajugo Include the initial migrations as part of this pr. You should amend your commit message to include you run initial migrations for the database.

File: authors/settings.py
Comment (archibishop) [inline authors/settings.py]: You should include defaults for all these variables. 
```suggestion
        "NAME": os.environ.get("DB_NAME", "authors_haven_db"),
```

Comment (3Nakajugo) [reply authors/settings.py]: @archibishop  please check the changes.

**Review by mwinel — APPROVED** [review]
I think this is ready for a merge.

**Review by joelmugaya — APPROVED** [review]

**Review by KabohaJeanMark — APPROVED** [review]
Great work @3Nakajugo  on following the Engineering handbook guidelines and responding to @archibishop,  our TTL's requested changes as per running the migrations to enable deployment and including defaults for .env configurations for the db settings.

**Comment by archibishop** [comment]
Is it possible to use this `psycopg2-binary` as suggested in the warnings and check if it doesn't break anything. Then i should approve after this. Good work though.
Screen Shot 2019-04-24 at 10 21 02

**Comment by archibishop** [comment]
@3Nakajugo Squash ur commits into one so i can merge.

**Review by archibishop — APPROVED** [review]
LGTM @3Nakajugo I am merging this. Good work here.

**Pull request merged**
